module/os/linux/zfs/abd.c

```diff
@@ -207,6 +207,14 @@ static abd_stats_t abd_stats = {
 #define	abd_for_each_sg(abd, sg, n, i)	\
 	for_each_sg(ABD_SCATTER(abd).abd_sgl, sg, n, i)
 
+#define	ABD_MULTI(abd)		(abd->abd_u.abd_multi)
+#define	abd_is_multi(abd)	((abd->abd_flags & ABD_FLAG_MULTI_LIST) != 0)
```

Comment:
We should consider splitting the new multiple ABD type in to its own commit and PR.  This functionality is useful by itself to eliminate the copy from `vdev_queue_aggregate()` and could be more easily reviewed and merged as a smaller self contained change.

Reply:
This would simple enough for me to do. I will go ahead and create a separate PR for this.